neo4j/result_with_context.go

```diff
@@ -45,8 +45,9 @@ type ResultWithContext interface {
 	Record() *Record
 	// Collect fetches all remaining records and returns them.
 	Collect(ctx context.Context) ([]*Record, error)
-	// Single returns one and only one record from the stream.
-	// If the result stream contains zero or more than one records, error is returned.
+	// Single returns the only remaining record from the stream.
+	// If none or more than one record are left, an error is returned.
+	// The result is fully consumed after this call and its summary is immediately available when calling Consume.
```

Comment:
Could `Consume` use be link? https://go.dev/doc/comment#doclinks

Reply:
So I tried your suggestion with `godoc` and it does not seem to link anywhere. Even bumping the project to Go 1.19 does not seem to do the trick.
Goland autolinks with both the current and suggested version.

Reply:
Let's skip the autolinking for now. If I manage to get it work, I'll have to update more than this API anyway.